README.md

```diff
@@ -1,16 +1,9 @@
 # elastic-package
 
-The elastic-package is a command line tool, written in Go, used for developing Elastic packages.
+`elastic-package` is a command line tool, written in Go, used for developing Elastic packages. It can help you lint, format, 
+test, build, and promote your packages. Learn about each of these and other features in [_Commands_](#commands) below.
 
-*For experimental use only*
-
-## Features
-
-TODO
-
-## Supported package types
-
-* [Elastic Integrations](https://github.com/elastic/integrations)
+Currently, `elastic-package` only supports packages of type [Elastic Integrations](https://github.com/elastic/integrations).
 
 ## Getting started
 
```

Comment:
If you are cleaning up the README, it might be a good option to refresh the Getting started as well. Currently, with `go get` we will loose the information about the version, so maybe it's better to replace it with `git clone` and `make build`.

We can also mention that the tool is go-gettable (but it will loose information about revision).

Reply:
Good point — I will clarify this in the README.

However, in the future (not for this PR), I hope we can have a single command to install `elastic-package` with all it's features. It seems weird from a user's perspective that the `version` command only works if you checkout the source and build from it (but I understand why that is the case currently). In the future, maybe we can think of committing a `version.go` file that's updated every so often via `make build` and then `elastic-package version` reads from this file or something like that.

Reply:
Updated Getting Started section in efe6362dd89696e60eb48eb4dd845a67581eea76.